ANNOUNCE

```diff
@@ -9,33 +9,27 @@ Among other data, cclib extracts:
     * information about atomic and molecular orbitals
     * vibrational modes, excited states and transitions
     * charges, electrostatic moments and polarizabilities
-
 (For a complete list see https://cclib.github.io/data.html).
-
 cclib also provides some calculation methods for interpreting the electronic properties of molecules such as:
-
     * Mulliken, Lowdin, Stockholder type, Bader and other population analyses
     * Overlap population analysis
     * Mayer's bond orders
-
 (For a complete list see https://cclib.github.io/methods.html).
-
 For information on how to use cclib, see our documentation at https://cclib.github.io.
-
 If you need help, find a bug, want new features or have any questions, please send an email to our mailing list:
     https://lists.sourceforge.net/lists/listinfo/cclib-users
-
 If your published work uses cclib, please support its development by citing the following article:
     N. M. O'Boyle, A. L. Tenderholt, K. M. Langner, cclib: a library for package-independent computational chemistry algorithms, J. Comp. Chem. 29 (5), 839-845 (2008)
 You can also specifically reference this version of cclib as:
-    Eric Berquist, Geoffrey Hutchison, Karol M. Langner, Noel M. O'Boyle, Adam L. Tenderholt, Shiv Upadhyay. Release of cclib version 1.7. 2020. https://dx.doi.org/10.5281/zenodo.4420433
-
+    Eric Berquist, Geoffrey Hutchison, Karol M. Langner, Noel M. O'Boyle, Adam L. Tenderholt, Shiv Upadhyay. Release of cclib version 1.7. 2020. https://doi.org/10.5281/zenodo.4420433
```

Comment:
It looks like we have a chicken-and-egg problem with minting a new (Zenodo) DOI and updating this file.  We will have to update it again after tagging and uploading to Zenodo.

Reply:
Indeed, always frustrated me before. Thanks for merging.